stable/testlink/values.yaml

```diff
@@ -17,6 +17,14 @@ image:
   # pullSecrets:
   #   - myRegistrKeySecretName
 
+
+## List of nodeSelectors to limit where testlink can run
+## i.e
+##
+## nodeSelector:
+##   kops.k8s.io/instancegroup: my-ig
+nodeSelector: {}
```

Comment:
@jaredallard could you please add this new parameter to the **README.md** file as we do for every parameter?